sdk/storage/Azure.Storage.Queues/src/Generated/AzureQueueStorageModelFactory.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Storage.Queues.Models;
+
+namespace Azure.Storage.Queues
+{
+    /// <summary> Model factory for AzureQueueStorage read-only models. </summary>
+    public static partial class AzureQueueStorageModelFactory
```

Comment:
Need to be reconciled with https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Queues/src/Models/QueuesModelFactory.cs

Reply:
Could the generator define a new attribute like `CodeModelAttribute` where, if provided, you use that name? That would also help resolve [one of my comments](https://github.com/Azure/azure-sdk-for-net/pull/21225#discussion_r636434212).